test/test_operators.py

```diff
@@ -157,6 +157,41 @@ def test_grad_operator(actx_factory, tpe, dim, order, sym_test_func_factory):
     """
     import pyopencl as cl
     from grudge.array_context import PyOpenCLArrayContext
+    from meshmode.mesh.processing import rotate_mesh_around_axis
+    from grudge.dt_utils import h_max_from_volume
+    from mirgecom.simutil import componentwise_norms
+    from arraycontext import flatten
+    from mirgecom.operators import grad_operator
+    from meshmode.mesh.processing import map_mesh
+
+    def add_wonk(x: np.ndarray) -> np.ndarray:
+        wonk_field = np.empty_like(x)
+        if len(x) >= 2:
+            wonk_field[0] = (
+                1.5*x[0] + np.cos(x[0])
+                + 0.1*np.sin(10*x[1]))
+            wonk_field[1] = (
+                0.05*np.cos(10*x[0])
+                + 1.3*x[1] + np.sin(x[1]))
+        else:
+            wonk_field[0] = 1.5*x[0] + np.cos(x[0])
+
+        if len(x) >= 3:
+            wonk_field[2] = x[2] + np.sin(x[0] / 2) / 2
+
+        return wonk_field
+
+    do_wonk_test = False
+
+    do_rotation_test = False
+    if dim > 1:
+        do_rotation_test = True
+        if dim == 2:
+            rotation_axes = [[0, 0, 1]]
+        else:
+            rotation_axes = [[0, 0, 1],
+                             [0, 1, 1],
+                             [1, 1, 1]]
```

Comment:
```suggestion
            rotation_axes = [np.array([0, 0, 1])]
        else:
            rotation_axes = [np.array([0, 0, 1]),
                             np.array([0, 1, 1]),
                             np.array([1, 1, 1])]
```
(`rotate_mesh_around_axis` docs say it takes `ndarray`, so I guess the current version just works by accident?)